test/bin/_head.sh

```diff
@@ -30,14 +30,23 @@ subpath_run_cmd () {
 
 eval_or_exit() {
     local RET_CODE
+    local COMMAND
+
+    COMMAND="`which istanbul` cover _mocha -- --compilers js:babel/register --reporter spec --ui tdd"
```

Comment:
@vcernomschi Please rename this var to smth more descriptive (e.g. TEST_DEBUG_CMD), and move it to "#Run test-debug to show error" if branch if possible.